Automattic/wp-calypso
# Seller Experience: Update DesignPicker to allow it to hide the listing of categories

#### Changes proposed in this Pull Request

* Add a CSS class to the DesignPicker component instance for the sell intent so we can use CSS to hide the categories with a specific class; this is less invasive than altering the component itself for this one circumstance.
* Only show "Store" designs for the `sell` intent.

**Video demo**

https://user-images.githubusercontent.com/2124984/158459381-ef3d754f-8ab2-4116-823b-09e8a9a10b67.mov



#### Testing instructions

* Switch to this PR, navigate to /start
* Create a site with the sell intent, choosing "Sell" and "Simple"
* You should be brought to the design picker with only five themes showing and the category filters hidden (but left sidebar area still visible)
* Go back and try other intents; you should be shown the design picker with all themes and the category filters.

Fixes #61921
